test/testdata/input/doctype-title.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<script src="../../../resources/js-test.js"></script>
+<script>
+description("Testing to see if title under DOCTYPE in transformed HTML");
```

Comment:
would probably be good to test a case with multiple descriptions (where one is used for title and others are dropped)
..somewhere, not necessarily in this file.

Reply:
That's a transform.js feature, which I test once with 2 description calls in multiline_file.js  in testTransform.js. Should we add more tests than that?

Reply:
Yeah, I noticed `multiline_file.js` but then `transformed_multiline_file.js` has no title (cuz it's js?)

Reply:
injectScripts takes the title as a parameter. As long as transform.js only returns one title, it's impossible for the HTML to contain more than one <title> due to multiple description() calls.

Reply:
Ok, so for multiline_file.js, the expected title is in the test rather than in the reference file -- sorry for missing that.

The case I was thinking of testing was "multiple description() calls are transformed into a single <title> tag where the title is the 1st description()". Which I don't think we're explicitly checking?

Reply:
Discussed offline, tests cover the case I was after.